libraries/AP_HAL_ChibiOS/hwdef/scripts/chibios_hwdef.py

```diff
@@ -767,6 +767,15 @@ def get_flash_pages_sizes():
           mcu_series.startswith('STM32G4') or
           mcu_series.startswith('STM32L4')):
         return [ 2 ] * (get_config('FLASH_SIZE_KB', type=int)//2)
+    elif mcu_series.startswith('STM32L4'):
```

Comment:
I don't see how this ever gets hit give the STM32L4 above

Reply:
Yes Noted. Thanks !!! Mistake during upload.